### ISSUE TITLE
cit_magic3.py doesn't recognized files within the current directory with -ID flag

### ISSUE THREAD

**Issue by Swanson-Hysell** [PmagPy/PmagPy#136]
This code throws an error, not recognizing the specimen files
```
cit_magic3.py -f AP4-.sam -loc 'Agate Point' -A -spc 1 -ncn 3
```
The error looks like this:
```
IOError: [Errno 2] No such file or directory: '/AP1-1a'
```

This workaround works at the moment:
```
cit_magic3.py -f AP4-.sam -loc 'Agate Point' -A -spc 1 -ncn 3 -ID .
```

**Comment by Swanson-Hysell** [comment]
Another issue: In data model 3, there is no site column in specimens, there is simply a sample column.

**Comment by Swanson-Hysell** [comment]
A couple more issues with cit_magic3.py that I will keep in this issue and reopen it:

- [ ] ```meas_n_orient``` is currently set to output 1. For us this number is almost always 8. I think it makes sense to put in a flag related to this (that can be a dialog within the gui) and to set the default to 8 if nothing is entered (rather than a default of 1).
- [ ] ```treat_dc_field_theta``` currently has a default of -90. If nothing is entered, I think it is simpler if the default is set to 0.

**Comment by Caoimhinmg** [comment]
Well as things stand now meas_n_orient is hardcoded to '1' I can hardcode it '8' or put in an option for meas_n_orient in my new script which is almost done, just testing it. Just fixed the dc_field_theta in the new script, will commit when I finish testing.

**Comment by Swanson-Hysell** [comment]
putting in an option for meas_n_orient makes sense with a default to 8.

**Comment by ltauxe** [comment]
this should be made part of the CIT files because some labs only do 4 positions and some do 8.  guess that boat sailed?

**Comment by Swanson-Hysell** [comment]
We write a separate log file that records every single measurement so that information is certainly recorded. One approach could be to also import these separate log files. However, since the code doesn't currently talk to those log files (one is produced for every measurement step) it would be a simpler solution to have the user enter the meas_n_orient number.

**Comment by Caoimhinmg** [comment]
Okay @Swanson-Hysell I have made a flag for meas_n_orient and defaulted it to 8 as requested. This ability is currently only available in the new pure 3.0 script I just pushed which is in programs/3_conversion_scripts/ in the main PmagPy repo. This is where I plan to put these native 3.0 scripts while I'm working on producing them so they don't clash with the old scripts and because I'm running out of names with a cit_magic and a cit_magic3 already taken. I also greatly expanded documentation of cit_magic. Let me know if you run into any troubles.
